pkg/apis/pipeline/v1alpha1/workspace_types.go

```diff
@@ -24,5 +24,5 @@ import (
 type WorkspaceDeclaration = v1alpha2.WorkspaceDeclaration
 
 // WorkspaceBinding maps a Task's declared workspace to a Volume.
-// Currently we only support PersistentVolumeClaims and EmptyDir.
+// Currently we only support PersistentVolumeClaims, EmptyDir and ConfigMap.
```

Comment:
I wonder if we'd be better off removing this comment so we don't have it fall out of date later :thinking: